src/Volume.jsx

```diff
@@ -28,7 +28,7 @@ export const Volume = ({ id, item, onChange, idx, removeitem, additem, options,
                 <Checkbox id={id + "-mode"}
                     label={_("Writable")}
                     isChecked={item.mode == "rw"}
-                    onChange={value => onChange(idx, 'mode', value ? "rw" : "ro")} />
+                    onChange={(_event, value) => onChange(idx, 'mode', value ? "rw" : "ro")} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-122810-fbb171ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)